its/ruling/src/test/resources/expected_extended/indico/python-S930.json

```diff
@@ -0,0 +1,5 @@
+{
+'indico:indico/util/string.py':[
+475,
```

Comment:
This is a FP due to the fact that the [latest](https://bleach.readthedocs.io/en/latest/clean.html#bleach.clean) version of the method doesn't allow for a `styles` parameter, but [previous](https://bleach.readthedocs.io/en/v4.0.0/_modules/bleach.html#clean) ones do.